lib/src/platform_client.dart

```diff
@@ -88,8 +86,7 @@ class PlatformClient {
       'phoneNumber': phoneNumber,
     });
 
-    Map<String, dynamic> response =
-        await _httpPost('/api/smartapp/verify/confirm', body);
+    Map<String, dynamic> response = await _httpPost('/api/smartapp/verify/confirm', body);
```

Comment:
Did you know that in IDEA you can select to reformat only the touched lines to avoid large PRs? 
<img width="1485" alt="Screenshot 2023-10-05 at 11 43 42 AM" src="https://github.com/aira/flutter_aira/assets/58491416/e6f6d949-33ff-4e8e-a911-f15c45c1aa14">

Reply:
I had forgotten about that one! thanks!